presto-native-execution/presto_cpp/main/http/tests/HttpTest.cpp

```diff
@@ -234,6 +258,154 @@ TEST_P(HttpTestSuite, timedOutRequests) {
   wrapper.stop();
 }
 
+TEST_P(HttpTestSuite, httpRequestTimeout) {
```

Comment:
Can you split the test into two? One is transaction timeout and the other is connect timeout? What's the difference in error message or error code? Thanks!

Reply:
Can we have them in one but change the name to httpTimeout instead of httpRequestTimeout. I've added comments below to indicate the connect/request timeout.